package.json

```diff
@@ -1,7 +1,12 @@
 {
   "name": "altare-tcg",
   "version": "0.0.1",
+  "type": "module",
```

Comment:
had this as part of attempt to get `import A11yDialog from 'a11y-dialog'` to work. Don't know if it's needed

Reply:
Only needed to use ES6 module syntax for imports. Not needed right now, but doesn't hurt either.